sha/.gitignore

```diff
@@ -0,0 +1,4 @@
+.DS_Store
+Cargo.lock
+methods/guest/Cargo.lock
```

Comment:
I think lock files should probably be commited for apps (just not for libs): 
https://doc.rust-lang.org/cargo/faq.html#why-do-binaries-have-cargolock-in-version-control-but-not-libraries